bundles/framework/announcements/resources/scss/announcementsplugin.ol.scss

```diff
@@ -26,6 +26,7 @@ div.announcements-content {
 }
 
 div.mapplugin.announcements {
+    padding: 10px 10px 0 10px;
```

Comment:
Does the new editor generate HTML that now requires these padding/align changes (Should the banner/popups be changed as well) or is this some unrelated tuning for the plugin?

Reply:
No, the announcements plugin is not yet in line with the other plugins in regards of styling, so this would be a temporary quick fix so the plugins above this wont touch.